docs/dev/plugins.rst

```diff
@@ -211,6 +211,22 @@ The events currently available are:
 * *import_begin*: called just before a ``beet import`` session starts up.
   Parameter: ``session``.
 
+* *trackinfo_received*: called after meta data for a track item has been fetched
+  from disparate sources, such as MusicBrainz. Gives a developer the option to
+  intercept the fetched TrackInfo object. Can be used to modify tags on a ``beet
+  import`` operation or during later adjustments, such as ``mbsync``. Can be
+  slow, as event is fired for any fetched possible match *before* user or
```

Comment:
Does "Can be slow" mean "During a normal, multi-threaded import, this event is invoked asynchronously with the user interaction, so slow handlers will not interrupt the flow."? If so, maybe that additional level of detail would be good here.